updated colorization scheme for packages

Robert-Krajcik opened the pull request:
Resolves issue #112 
Changes below.
Note I was able to use the CSS *order* property so it sorts on order not the background color hashtag

```{r, eval=FALSE}
  colfunc <- colorRampPalette(c(low_risk_color, "gold", high_risk_color))
  
  as.datatable(
    formattable(
      values$db_pkg_overview,
      list(
        score = formatter(
          "span",
          style = x ~ style(display = "block",
                            "border-radius" = "4px",
                            "padding-right" = "4px",
                            "font-weight" = "bold",
                            "color" = "white",
                            "order" = x,
                            "background-color" = csscolor(
                              colfunc(100)[round(as.numeric(x)*100)]))),
```
![image](https://user-images.githubusercontent.com/75215958/112207004-fe4e0200-8bec-11eb-892d-2ce9a0cf1e3f.png)
![image](https://user-images.githubusercontent.com/75215958/112207054-10c83b80-8bed-11eb-965f-bbf8ecefb654.png)

And the color remains the same regardless of how many/few packages are loaded.
![image](https://user-images.githubusercontent.com/75215958/112205460-2dfc0a80-8beb-11eb-93bd-6263c5d53527.png)

AARON-CLARK commented on Server/db_dash_screen.R:
```diff
@@ -44,42 +46,27 @@ output$db_pkgs <- DT::renderDataTable({
                             "background-color" = 
                               ifelse(x == "High Risk", high_risk_color,
-                                     ifelse(x == "Medium Risk", med_risk_color,
+                                     ifelse(x == "Medium Risk", "yellow",
```
Should this also be "gold" to match the middle color used for scores?

AARON-CLARK reviewed (changes requested):
Hi @Robert-Krajcik, just one suggestion below:

Robert-Krajcik commented:
Yeah we should be consistent here.
I'll update it

Robert-Krajcik commented:
Updated, @AARON-CLARK 
I changed the color palette slightly and am now using low_, med_, high_ risk names
```{r, eval=FALSE}
  # https://www.rapidtables.com/web/color/html-color-codes.html
  low_risk_color  <- "#228B22"  # forest green
  med_risk_color  <- "#FFD700"  # gold
  high_risk_color <- "#B22222"  # firebrick
  
  colfunc <- colorRampPalette(c(low_risk_color, med_risk_color, high_risk_color))
  # plot(rep(1,100),col=colfunc(100),pch=19,cex=3)
```
![image](https://user-images.githubusercontent.com/75215958/112319210-38b6ae00-8c84-11eb-9cd7-deb719743ab3.png)

AARON-CLARK reviewed (approved):
LGTM!
